### PR TITLE
Update macos.md - M1 arch

### PR DESCRIPTION
**Pull request by tboulis**
Add instructions for Macs with M1 chips

### CODE REVIEW

File: site/content/contribute/desktop/developer-setup/macos.md [hunk 1]
```diff
@@ -11,4 +11,10 @@
     ```sh
     echo 'export PATH="/usr/local/opt/node@16/bin:$PATH"' >> ~/.zshrc
     source ~/.zshrc
-    ```
\ No newline at end of file
+    ```
+    
+---
+For Mac with Apple Chip (M1 architecture):
```
Comment (neflyte) [inline site/content/contribute/desktop/developer-setup/macos.md:17]:
```suggestion
For machines using the M1 architecture:
```

File: site/content/contribute/desktop/developer-setup/macos.md [hunk 2]
```diff
@@ -11,4 +11,10 @@
...
+- Enable [Rosetta](https://support.apple.com/en-us/HT211861)
+- Use the latest NodeJS@16 (>=v16.16)
```
Comment (neflyte) [inline site/content/contribute/desktop/developer-setup/macos.md:19]:
```suggestion
2. Use the latest NodeJS@16 (>=v16.16)

```

File: site/content/contribute/desktop/developer-setup/macos.md [hunk 3]
```diff
@@ -11,4 +11,10 @@
...
+- Update [xcode](https://developer.apple.com/xcode/) to latest version (>=v13.4.1)
```
Comment (neflyte) [inline site/content/contribute/desktop/developer-setup/macos.md:20]:
```suggestion
3. Update [xcode](https://developer.apple.com/xcode/) to latest version (>=v13.4.1)

```

**Review by neflyte — CHANGES_REQUESTED** [review]

**Comment by neflyte** [comment]
Thanks for the contribution, @tboulis. I've left a few suggestions to bring this change in line with the current documentation format.

**Review by devinbinnie — APPROVED** [review]

**Review by neflyte — APPROVED** [review]

**Comment by M-ZubairAhmed** [comment]
i am not sure if we should mention latest node 16 though i remember having problems. At the very least stick to the version exactly which is the web app ?

**Comment by cwarnermm** [comment]
@tboulis - Are you open to address the merge conflicts on this PR?

**Pull request merged**